rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -529,6 +543,30 @@ def test_exec(config, ssh_con):
                                 raise TestExecError(
                                     "num shards are same after processing resharding"
                                 )
+                        if config.sync_disable_and_enable:
+                            config.objects_count = config.objects_count + 10
+                            config.mapped_sizes = utils.make_mapped_sizes(config)
+                            for oc, size in list(config.mapped_sizes.items()):
+                                config.obj_size = size
+                                s3_object_name = utils.gen_s3_object_name(
+                                    bucket_name_to_create, oc
+                                )
+                                log.info("s3 object name: %s" % s3_object_name)
+                                s3_object_path = os.path.join(
+                                    TEST_DATA_PATH, s3_object_name
+                                )
+                                log.info("s3 object path: %s" % s3_object_path)
+                                reusable.upload_object(
+                                    s3_object_name,
+                                    bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+                            op = utils.exec_shell_cmd(
+                                f"radosgw-admin bucket sync enable --bucket {bkt}"
+                            )
```

Comment:
@udaysk23 could we call the function check_bucket_sync_status here and validate the status